.travis.yml

```diff
@@ -14,8 +14,8 @@ env:
   - secure: Dk11Yrq0XT+VXXncqhyV+g0PDV+teFWobAigDpsncNKFtSth37Eoi6lrPVKT9S5b5qh+SVx7h7D3VSpMPoIAXuaeG62yn7e1RegCFk19xJB8aGsooVhwdlQk8yiCHe/1ITr6C95Ftj4QQT1k29inAsjvAqLPU229EwN+In4nAZw2S8qMR27FqCd6HvdrbwaM5PYVHvgANgBkL3awMmIm4On5wSqOdOukj8L7LztUA87YKfgUsOln1ERI+nN6fbN0FYdTFEWwG56HNhNvOSv5Gt3os92r47r0VFghIRrJOjwEWE2KFsEs43DYemrJBlgu0OCvk9A3gYr0vUeVxgT6LH5zZUTavK2ew6sZmUBpJLag9oezC1mjp3c8s90eRZX5eCaJ0Dyt4us9z3B96SEHg3lF3PZDq0xloeuxa8dgJp4HJzQi7HNn8sMezFE2DMeZ2x1zOuNbyiI6ZmDMCp54me1Z7MOk+MCu6q/8JAPpfEeaFPNVt5qYolI0pYzHURU1RVMzVN2sSaFMEnyBR6W/f5MoIHYGZFBygTPHZVgjkdpniKS+Meo6RrmXyrmDQMAc08bDjdepiQO51QP8h5BKXKkTrIa13XvOpVGkmbfBzoYC6RzVRZkovERQh2DSqMG+vclnDFzcBZ8xEqesAEp2c8tlSamTLu6wyeC0gi0VAxo=
 notifications:
   slack:
-    if: branch = master AND type = push
+    if: branch = master AND (type = push OR type=cron)
```

Comment:
What is the `cron` for?

Reply:
So that if the weekly build fails (triggered by a chron) we get notified. By default the slack plugin messages for every build on every branch, we are limiting it to when a PR is merged into master and when the weekly cron job runs against master. We are not going to notify the slack channel if a feature branch build fails or a PR opened/updated build fails.